### PR TITLE
bt@3.6.1: Fix persist

### PR DESCRIPTION
**Pull request by neoOpus**
attempt to make the persistent works.

Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/bt.json [hunk 1]
```diff
@@ -21,7 +21,11 @@
             "Browser Tamer"
         ]
     ],
-    "persist": "bt.ini",
+    "persist": [
+        "config.ini", 
+        "clearurls_db.json",
+        "hit_log.csv"
+    ]
```
Comment (HUMORCE) [inline bucket/bt.json:28]:
```suggestion
    ],
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]
syntax

**Comment by liny01-nbsa** [comment]
Is this PR abandoned? I can create a new PR for this change if appropriate. It saves people the trouble for any manual post fixes.

**Comment by neoOpus** [comment]
> Is this PR abandoned? I can create a new PR for this change if appropriate. It saves people the trouble for any manual post fixes.

Apologies for the confusion, but I need guidance on why this PR interests you. It fixes a bug, but I've forgotten the exact details of what it does.

**Comment by neoOpus** [comment]
Oh now I recall, yeah I guess you came here after you've lost the config after the new update.

Please do what you think is more appropriate to fix this issue once for all for everyone!

**Comment by liny01-nbsa** [comment]
Hi @neoOpus , nice to have you here. Looks @HUMORCE just need you to add a comma by the end of array close. If you still have the repo around, maybe you can still fix it, and have the PR merged as your change.

**Comment by liny01-nbsa** [comment]
@neoOpus I used your change and created a new PR #12733. 😊

@HUMORCE , please check it out, and if it works, it closes both.

**Comment by aliesbelik** [comment]
Fixed in #12733, closing this.

**Pull request closed without merging**